newrelic/src/main/java/com/cerner/beadledom/newrelic/NewRelicApiImpl.java

```diff
@@ -0,0 +1,15 @@
+package com.cerner.beadledom.newrelic;
+
+/**
+ * Default implementation that delegates to the New Relic API.
+ *
+ * @author Nathan Schile
+ * @since 2.8
+ */
+public class NewRelicApiImpl implements NewRelicApi {
+
+  @Override
+  public void addCustomParameter(String key, String value) {
+    com.newrelic.api.agent.NewRelic.addCustomParameter(key, value);
```

Comment:
I think this can be imported now since the name of this class is different.

Reply:
Yup, https://github.com/cerner/beadledom/pull/93/commits/96821090e56ce3393b95fa790babdeffe1d0e6cb